content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -1,51 +1,66 @@
+based on a common template. You can use this approach to process batches of work in
+parallel.
 
+For this example there are only three items: _apple_, _banana_, and _cherry_.
+The sample Jobs process each item simply by printing a string then pausing.
+
+See [using Jobs in real workloads](#using-jobs-in-real-workloads) to learn about how
+this pattern fits more realistic use cases.
 {{% /capture %}}
 
+{{% capture prerequisites %}}
 
-{{% capture body %}}
+You should be familiar with the basic,
+non-parallel, use of [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/).
 
-## Basic Template Expansion
+{{< include "task-tutorial-prereqs.md" >}}
 
-First, download the following template of a job to a file called `job-tmpl.yaml`
+For basic templating you need the command-line utility `sed`.
 
-{{< codenew file="application/job/job-tmpl.yaml" >}}
+To follow the advanced templating example, you need a working install of
+[Python](https://www.python.org/), and the Jinja2 template
+library for Python.
 
-Unlike a *pod template*, our *job template* is not a Kubernetes API type.  It is just
-a yaml representation of a Job object that has some placeholders that need to be filled
-in before it can be used.  The `$ITEM` syntax is not meaningful to Kubernetes.
+Once you have Python set up, you can install Jinja2 by running:
+```shell
+pip install --user jinja2
+```
+{{% /capture %}}
 
-In this example, the only processing the container does is to `echo` a string and sleep for a bit.
-In a real use case, the processing would be some substantial computation, such as rendering a frame
-of a movie, or processing a range of rows in a database.  The `$ITEM` parameter would specify for
-example, the frame number or the row range.
 
-This Job and its Pod template have a label: `jobgroup=jobexample`.  There is nothing special
-to the system about this label.  This label
-makes it convenient to operate on all the jobs in this group at once.
-We also put the same label on the pod template so that we can check on all Pods of these Jobs
-with a single command.
-After the job is created, the system will add more labels that distinguish one Job's pods
-from another Job's pods.
-Note that the label key `jobgroup` is not special to Kubernetes. You can pick your own label scheme.
+{{% capture steps %}}
 
-Next, expand the template into multiple files, one for each item to be processed.
+## Create a template Job
+
+First, download the following template of a Job to a file called `job-tmpl.yaml`
+
+{{< codenew file="application/job/job-tmpl.yaml" >}}
 
 ```shell
-# Download job-templ.yaml
+# Use curl to download job-templ.yaml
 curl -L -s -O https://k8s.io/examples/application/job/job-tmpl.yaml
+```
+
+This *job template* is not yet a valid Kubernetes {{< glossary_tooltip text="manifest" term_id="manifest" >}}.
+Instead that template is a YAML representation of a Job object with some placeholders
```

Comment:
nit: Lines 54-55, Could clean up a bit.
The `job-tmpl.yaml` manifest represents a Kubernetes Job object with some placeholders for field values.

Reply:
I prefer my wording to be honest. I really want to make clear that `job-tmpl.yaml` is not intended to be used as a manifest directly.